zyte_common_items/components.py

```diff
@@ -148,3 +162,122 @@ class Metadata(Item):
     #: `0`. When there is no complete certainty, the value could be anything in
     #: between (e.g. `0.96`).
     probability: Optional[float] = None
+
+
+@attrs.define(kw_only=True)
+class BusinessPlaceMetadata(Metadata):
+    """Data extraction process metadata.
+
+    See :class:`BusinessPlace.metadata <zyte_common_items.items.BusinessPlace.metadata>`.
+    """
+
+    #: The search text the place was found with.
+    searchText: Optional[str] = None
+
+
+@attrs.define(kw_only=True)
+class Address(Item):
+    """Address item."""
+
+    #: The raw address information, as it appears on the website.
+    addressRaw: Optional[str] = None
+
+    #: The street address of the place.
+    streetAddress: Optional[str] = None
+
+    #: The city the place is located in.
+    addressCity: Optional[str] = None
+
+    #: The locality to which the place belongs.
+    addressLocality: Optional[str] = None
+
+    #: The region of the place.
+    addressRegion: Optional[str] = None
+
+    #: The country the place is located in.
+    #:
+    #: The country name or the two-letter ISO 3166-1 alpha-2 country code.
+    #: https://en.wikipedia.org/wiki/ISO_3166-1
+    addressCountry: Optional[str] = None
+
+    #: The postal code of the address.
+    postalCode: Optional[str] = None
+
+    #: The auxiliary part of the postal code.
+    #: It may include a state abbreviation or town name, depending on local standards.
```

Comment:
```suggestion
    #: The auxiliary part of the postal code.
    #:
    #: It may include a state abbreviation or town name, depending on local standards.
```

An empty line is needed to cause separate paragraphs in the rendered docs.